src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy_v2.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+"""
+Organizations Policy module used throughout the ADF.
+"""
+
+import glob
+import os
+import ast
+import json
+import boto3
+
+from logger import configure_logger
+from organization_policy_campaign import (
+    OrganizationPolicyApplicationCampaign,
+)
+from errors import ParameterNotFoundError
+
+
+from organizations import Organizations
+
+from organization_policy_schema import OrgPolicySchema
+
+LOGGER = configure_logger(__name__)
+REGION_DEFAULT = os.getenv("AWS_REGION")
+DEFAULT_ADF_POLICIES_DIR = "./adf-policies"
+
+
+class OrganizationPolicy:
+    adf_policies_dir: str
+
+    def __init__(self, adf_policies_dir=None):
+        self.adf_policies_dir = adf_policies_dir or DEFAULT_ADF_POLICIES_DIR
+
+    @staticmethod
+    def _find_all_legacy_policies(policy):
+        _files = list(
+            glob.iglob(
+                f"./adf-bootstrap/**/{policy}.json",
+                recursive=True,
+            )
+        )
+        return [f.replace("./adf-bootstrap", ".") for f in _files]
+
+    def _find_all_polices(self, policy):
+        _files = list(
+            glob.iglob(
+                f"{self.adf_policies_dir}/{policy}/*.json",
+                recursive=True,
+            )
+        )
+        return [f.replace(f"{self.adf_policies_dir}", ".") for f in _files]
+
+    @staticmethod
+    def _trim_scp_file_name(policy):
+        return policy[1:][:-8] if policy[1:][:-8] == "/" else policy[2:][:-9]
+
+    @staticmethod
+    def _trim_tagging_policy_file_name(policy):
+        return policy[1:][:-19] if policy[1:][:-19] == "/" else policy[2:][:-20]
+
+    @staticmethod
+    def _is_govcloud(region: str) -> bool:
+        """
+        Evaluates the region to determine if it is part of GovCloud.
+
+        :param region: a region (us-east-1, us-gov-west-1)
+        :return: Returns True if the region is GovCloud, False otherwise.
+        """
+        return region.startswith("us-gov")
+
+    def get_policy_body(self, path):
+        with open(
+            f"{self.adf_policies_dir}/{path}", mode="r", encoding="utf-8"
+        ) as policy:
+            return json.dumps(json.load(policy))
+
+    def apply(
+        self, organizations, parameter_store, config
+    ):  # pylint: disable=R0912, R0915
+        status = organizations.get_organization_info()
+        if status.get("feature_set") != "ALL":
+            LOGGER.info(
+                "All Features are currently NOT enabled for this Organization, "
```

Comment:
If we are tasked to apply an SCP or Tagging Policy, maybe it would be better to raise this as a warning or even a fatal error. As it might go unnoticed otherwise. What do you think?